core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1135,6 +1136,71 @@ object TestUtils extends Logging {
   }
 
   /**
+   * Verifies that this ACL is the secure one. The
+   * values are based on the constants used in the
+   * ZooKeeper code base.
+   */
+  def isAclSecure(acl: ACL, sensitive: Boolean): Boolean = {
+    info(s"ACL $acl")
+    acl.getPerms match {
+      case 1 => !sensitive && acl.getId.getScheme.equals("world")
+      case 31 => acl.getId.getScheme.equals("sasl")
+      case _ => false
+    }
+  }
+
+  /**
+   * Verifies that the ACL corresponds to the unsecure one.
```

Comment:
It would be good to expand this a bit more. It seems that the definition if `unsecure` is a bit specific.